Training/Program.cs

```diff
@@ -1,7 +1,79 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
+﻿//Numbers to words or roman numbers
+
+using static System.Console;
+
+var key = ConsoleKey.Y;
+while (key == ConsoleKey.Y) {
+    Write ("Enter a Number : ");
+    if (!int.TryParse (ReadLine (), out int num) && num >= 0) {

```

Comment:
```suggestion
    if (!int.TryParse (ReadLine (), out int num) || num < 0) {

```
You negated the if but not the &&
If you enter the number as -1, your program hangs now.